frameworks/js-bindings/bindings/manual/network/XMLHTTPRequest.cpp

```diff
@@ -181,16 +182,21 @@ void MinXmlHttpRequest::handle_requestResponse(cocos2d::network::HttpClient *sen
     if (!response->isSucceed())
     {
         CCLOG("Response failed, error buffer: %s", response->getErrorBuffer());
+        if (statusCode == 0)
+        {
+            _errorFlag = true;
+            _status = 0;
+            _statusText.clear();
+            return;
```

Comment:
Hi, @seobyeongky , if `statusCode` is `0`, why don't we need to notify JS that there is an error triggered?
Here, we just return without any notifications to JS.